Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -1334,8 +1334,12 @@ extension Tensor {
 
   /// Returns `true` if the given scalar tensor is in the range `[-rank, rank)`.
```

Comment:
I realize this isn't part of your change, but a few things about the comment:
- Should “the given scalar tensor” be consider sufficient to imply the precondition?  I think it kinda buries the conditions for using the function in an adjective, so it might be better to have an explicit precondition clause.
- It's more descriptive to say “Returns `true` if `axis` is in…”
- “`[-rank, rank)`” implies that it's valid code. If you want to use that notation, I think you want “[`-rank`, `rank`)” but very frankly I think `-rank..<rank` is more appropriate here (and you can leave out  “the range”)
- It's even more descriptive to say something about what it returns that's less like code and more like its high-level meaning, e.g. something analogous to “refers to an axis of `self`”